src/H5Sselect.c

```diff
@@ -413,7 +413,7 @@ H5S_get_select_npoints(const H5S_t *space)
     Check whether the selection fits within the extent, with the current
     offset defined.
  USAGE
-    htri_t H5Sselect_void(dsid)
+    htri_t H5Sselect_valid(dsid)
         hid_t dsid;             IN: Dataspace ID to query
```

Comment:
I assume dsid here and spaceid in line 430 are the same.  Would it be better if they match?